browser/src/control/jsdialog/Widget.SidebarContainers.ts

```diff
@@ -67,6 +67,14 @@ JSDialog.panel = function (
 			{
 				type: 'toolitem',
 				command: expanderData.command,
+				aria: {
+					label: data.children[0].text
+						? _('More options for %name').replace(
```

Comment:
Please use {name} instead of %name, cf. bbe25379d5e23d1e0950d2cd8f5e209f3705334f